### PR TITLE
Adding Linter Files to Workflow Directory

### PR DESCRIPTION
**Pull request by JEFFKAY16**
In this branch, I did the following tasks:

1. Added an HTML file and an h1 tag.
2. Added the CSS file containing styles for the h1 tag.
3. Listing the test.md file into the .gitignore file so that it isn't listed in the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Hello-world
\ No newline at end of file
+This Hello Microverse project is my first project as a Microverse student. I will be setting up HTML and CSS linters alongside 3 files of which one is to be ignored by git.
```
Comment (DanteAlonsoHT) [inline README.md]: - [ ] For all your projects you will do in Microverse, you should make a descriptive and professional README file, feel free to use the [Template provided by Microverse](https://github.com/microverseinc/readme-template), also remember to use only the sections necessary according to this project, you can remove "Run tests", "Deployment".


- [Suggestion] Instead "feature-1" you can use "create_sidebar", "added_comments_section" which are more descriptives. (The feature branch name depends on what you did in the project)

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @JEFFKAY16, 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Improve the description in your Pull Request, explain what you did, what is the main scope, which files do you changed or added, what you set up in your project in this pull request, and so on.

_Check the comments under the review._ 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better_.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# Hello-world
```
Comment (SajjadAhmad14) [inline README.md:1]: Please add a professional README file. A README should have the steps to run the project locally. You may use [this](https://github.com/microverseinc/readme-template) tempelate from Microverse.

File: README.md [hunk 3]
```diff
@@ -1 +1,12 @@
...
\ No newline at end of file
+Hello Microverse Project
```
Comment (SajjadAhmad14) [inline README.md:1]: [ OPTIONAL ] Please follow GitHub flow. The main/master branch should be the default branch.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Jeff Kayombo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AshakaE — APPROVED** [review]
### Hi @JEFFKAY16  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**